build.gradle

```diff
@@ -108,6 +108,18 @@ ext.avroVersion = '1.7.7'
 ext.dropwizardMetricsVersion = '3.1.0'
 ext.findBugsVersion = '3.0.0'
 
+if (project.hasProperty('useHadoop2')) {
+  println "Using Hadoop 2 adaptor."
+  ext.gobblinHadoop = ":gobblin-hadoop2"
```

Comment:
Allows to implement things that are only valid for Hadoop 2, or implement features different for Hadoop 1 and Hadoop 2.